src/css/admin-settings.scss

```diff
@@ -1,49 +1,85 @@
-:root {
+@import "../content-helper/common/css/variables";
+@import "../content-helper/common/css/functions";
+
+.settings_page_parsely {
 	--padding-default: 15px;
-}
 
-#track-post-types {
-	max-width: 550px;
-	width: 100%;
-}
+	#track-post-types {
```

Comment:
`#track-post-types` styles have been re-arranged under a single element, with no real code changes. This is also true for `@media` CSS.